front/tests/pages/op-timetable-page-model.ts

```diff
@@ -194,10 +211,39 @@ class OperationalStudiesTimetablePage {
     }
   }
 
-  async verifyTimeStopsDatasheetVisibility(timeout = 30 * 1000): Promise<void> {
-    await this.timeStopsDatasheet.scrollIntoViewIfNeeded();
-    // Wait for the Times and Stops simulation datasheet to be fully loaded with a specified timeout (default: 30 seconds)
-    await expect(this.timeStopsDatasheet).toBeVisible({ timeout });
+  async verifyTimeStopsDataSheetVisibility(timeout = 60 * 1000): Promise<void> {
+    // Wait for the Times and Stops simulation dataSheet to be fully loaded with a specified timeout (default: 60 seconds)
+    await expect(this.timeStopsDataSheet).toBeVisible({ timeout });
+    await this.timeStopsDataSheet.scrollIntoViewIfNeeded();
+  }
+
+  // Click on Edit train button
```

Comment:
Are the comments necessary for all these functions ?

Reply:
Removed : The name of the functions is pretty clear